maliput/src/utility/mod.rs

```diff
@@ -87,9 +99,8 @@ pub fn get_obj_description_from_road_network(
 ) -> Result<String, Box<dyn Error>> {
     let output_directory = std::env::temp_dir().join("maliput");
     if !output_directory.exists() {
-        let _ = create_dir(&output_directory);
+        let _ = create_dir_all(&output_directory);
```

Comment:
We could remove this `if` statement as well.

Reply:
Argh, you're right!
Done!